index.html

```diff
@@ -13,7 +13,95 @@
   <![endif]-->
 </head>
 
+
 <body>
-  <h1>My Resume</h1>
+  <!-- .container>header>nav>a*4^^.main>section-top>.img+h1+p*2+.button^section-middle>.box>h2+p^.box>h2+ul>li*5^^^section-bottom>.table>h2+p^^footer -->
+  <div class="container">
+    <header>
+      <nav>
+        <a href="">Home</a>
+        <a href="">About</a>
+        <a href="">Portfolio</a>
+        <a href="">Contact</a>
+      </nav>
+    </header>
+    <main> 
+       <section-top>
+        <div class="box">
+          <img src="img\DJ.jpg" alt="Community DJ" class="dj">
+          <!-- <img src="img\DJ.jpg" alt="Community DJ"> -->
+          <!-- </div> -->
+          <h1>Gregory Urban - Full Stack Developer</h1>
+          <p>Bridging Technology with Business</p>
+          <p>Delighted to help, just sign up here!</p>
+          <button>Get Greg</button>
+        </div>
+      </section-top>
+      <br>
+
+      <section-middle>
+        <div class="box">
+          <h2>Why hire me?</h2>
+          <p>Dependable integrity</p>
+          <br>
+        </div>
+        <div class="box">
+          <h2>Hobbies</h2>
+          <ul>
+            <li>Golf</li>
+            <li>Skiing, Snowboarding, X-Country Skiing</li>
+            <li>Piano & Guitar</li>
+            <li>Community DJ</li>
+            <li>Learning a new Tech</li>
+          </ul>
+        </div>
+      </section-middle>
+      <br>
+      
+      <section-bottom>
+        
+            <h2>Work History</h2>
+            
+            <table style="width:80%">
+                <tr>
+                  <th>Years</th>
+                  <th>Company</th> 
+                  <th>Title</th>
+                  <th>Tech or Skill Used</th>
+                </tr>
+                <tr>
+                  <td>10</td>
+                  <td>Validas</td> 
+                  <td>VP of Enterprise Development</td>
+                  <td>SSQL, Excel VBA, QA, PM</td>
+                </tr>
+                <tr>
+                  <td>1</td>
+                  <td>Greendata</td> 
+                  <td>Front End QA and Maintenance</td>
+                  <td>Blockchain, VSC, Git</td>
+                </tr>
+                <tr>
+                  <td>1</td>
+                  <td>Lambda Bootcamp</td> 
```

Comment:
Don't call it a boot camp! The way this jumped out at me may be evidence that I'm drinking the koolaid. haha